DFC.Integration.AVFeed/DFC.Integration.AVFeed.Function.GetAVForSoc/ApprenticeshipVacancyFunction.cs

```diff
@@ -31,8 +33,9 @@ public MappedVacancySummary GetOutput()
             {
                 Vacancies = vacancySummaries,
                 SocCode = socMapping.SocCode,
-                SocMappingId = socMapping.SocMappingId,
-                AccessToken = socMapping.AccessToken
+                SocMappingId =  socMapping.SocMappingId,
+                AccessToken = (vacancySummaries.Any()) ? socMapping.AccessToken:null,
+                IsValidVacancy = (vacancySummaries.Any()) ?true:false
```

Comment:
Not correct logic, there will valid mapping but returns 0 vacancies at the time of call and can have vacancy added to it later etc. So checking for summary count as 0 is not a valid logic.